### PR TITLE
Ports aquariums and sushi

### PR DESCRIPTION
**Pull request by jakmak5**
Ports paradises fish tanks and sushi recipes.

Here is a webm of it in action

https://puu.sh/tJkcW/16db19a379.webm

You can make the boards for the fish tank and large fish tank via solder or order them from cargo

The eggs and fish care tools are bough from cargo in the aqua culture crate and the exotic fish eggs crate.

To add fish in just insert water and fish eggs to the tank and use the fish egg scoop to harvest any eggs that the fish lay. Re add the eggs to get more fish and use the net to pull fish out of the tank. Use the fish brush to clean the gunk off the tank to ensure happy and healthy fish.

Once harvested salmon and catfish can be gutted using any knife as so they could be used for making fish related recipes

### CODE REVIEW

**Comment by PJB3005** [comment]
You just reverted the supplypack name standardization.

**Comment by jakmak5** [comment]
yeah i noticed that and fixing it now

**Comment by yclatious** [comment]
A fish farm for easy fish and cargo can now have some swanky aquariums, good stuff, shame about no actual carp breeding for its recipes, but still great.

File: code/game/machinery/constructable_frame.dm [hunk 1]
```diff
@@ -1213,3 +1214,27 @@ obj/item/weapon/circuitboard/rdserver
...
+ * Fishtanks
+*/
+
+
+/obj/item/weapon/circuitboard/fishtank
+	name = "Circuit Board (Fishtank Filter)"
+	build_path = "/obj/machinery/fishtank/tank"
+	board_type = MACHINE
+	origin_tech = Tc_PROGRAMMING + "=1"
+	frame_desc = "Requires 5 sheet of glass."
```
Comment (kol1th) [inline code/game/machinery/constructable_frame.dm:1229]: sheets

**Comment by JustSumBody** [comment]
Did you
Forget to include the new files?

**Comment by kol1th** [comment]
also [Update Wiki]

**Comment by CptWad** [comment]
reeeeeeee
pm me if you need help updating the wiki

File: code/modules/fish/fish_items.dm [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+	desc = "A tiny net to capture fish with. It's a death sentence!"
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "net"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = 2
+	throw_speed = 3
+	throw_range = 7
+
+	suicide_act(mob/user)			//"A tiny net is a death sentence: it's a net and it's tiny!" https://www.youtube.com/watch?v=FCI9Y4VGCVw
```
Comment (ghost) [inline code/modules/fish/fish_items.dm:41]: You're gonna get an angry comic here if you do this.

Do /obj/item/weapon/fishtools/fish_net/suicide_act(mob/user) instead

**Comment by Zth--** [comment]
Not a big fan of those sprites

**Comment by CptWad** [comment]
👍 or 👎 
image

**Comment by Nienhaus** [comment]
@CptWad that has no shading and is the worst perspective.

**Comment by CptWad** [comment]
hey man sorry im trying to get better
this better?
image

**Comment by Skullyton** [comment]
@CptWad Hide that shit, before skowron calls in a nuke from his submarine

**Comment by CptWad** [comment]
I dont understand how do I make it better

image

**Comment by Shadowmech88** [comment]
Remove the dark outline, for one.

**Comment by Skullyton** [comment]
Lessen the outline, add in some inbetween between the outline and the internal, and get rid of the weird shading you did beforehand

**Comment by CptWad** [comment]
image
?

**Comment by Shadowmech88** [comment]
>pillow shading

**Comment by CptWad** [comment]
image

**Comment by Skullyton** [comment]
Oh god, no wonder it's a pain, you're spriting it in DM

Get Paint.net, it'll save you time, hassle, and sanity

**Comment by Nienhaus** [comment]
@Skullyton gimp for life ;)

**Comment by ihadtoregisterforthis** [comment]
Use Aseprite and get easy animations too

**Comment by Blithering** [comment]
Well at least someone else still uses DM

**Comment by CptWad** [comment]
thanks blithering

**Comment by gbasood** [comment]
\>salmon eggs
carp sushi rolls when

**Comment by jakmak5** [comment]
when you give me a sprite for it because i cant sprite for shit. Also if anyone has any ideas for any fish related recipes if you can get me a sprite i can put it in.

**Comment by CptWad** [comment]
this any better?
image

**Comment by AutismSpider** [comment]
So whats the hold up?

**Comment by CptWad** [comment]
right i have to upload it too
[memes.zip](https://github.com/d3athrow/vgstation13/files/767957/memes.zip)
also comes with slightly better wall tank and fixed the overloads

**Comment by jakmak5** [comment]
going to have to remake this PR standby

**Pull request closed without merging**